packages/anon-creds/src/models/exchange.ts

```diff
@@ -0,0 +1,104 @@
+// TODO: Maybe we can make this a bit more specific?
+export type WalletQuery = Record<string, unknown>
+
+export interface NonRevokedInterval {
+  from?: number | undefined
+  to?: number | undefined
+}
+
+export interface AnonCredsCredentialOffer {
+  schema_id: string
```

Comment:
Very very small knowledge on the spec and the changes, but is it not `schemaId` now? very unsure, but I thought I remembered it from somewhere...